app/views/doc/ext.html.erb

```diff
@@ -126,6 +126,13 @@
             By Ravishankar Somasundaram
           </p>
          </li>
+        <li>
+         <a href="https://www.packtpub.com/application-development/git-essentials-second-edition"><img src="https://www.packtpub.com/sites/default/files/B06881.png" width="113" height="140"></a>
+         <h4><a href="https://www.packtpub.com/application-development/git-essentials-second-edition</a>Git Essentials - Second Edition</a></h4>
```

Comment:
you are still not closing the <a> tag properly. have a look at my comment in the original PR https://github.com/git/git-scm.com/pull/1086#discussion_r156309196